### DISCUSSION TITLE
train YOLO data set on Windows, and run on Jetson Nano (Ubuntu)

### DISCUSSION THREAD

**Discussion by VladimirSirbu** [AlexeyAB/darknet#8451] [Q&A] [answered]
Is it possible to train YOLO data set on PC (for example Windows) and then run file.weights on Nvidia Jetson Nano ?

**Comment by ricchagas** [comment]
Yes, it,s my use case

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: VladimirSirbu ***@***.***>
Sent: Tuesday, April 5, 2022 5:08:30 PM
To: AlexeyAB/darknet ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AlexeyAB/darknet] train YOLO data set on Windows, and run on Jetson Nano (Ubuntu) (Discussion #8451)


Is it possible to train YOLO data set on PC (for example Windows) and then run file.weights on Nvidia Jetson Nano ?

**Comment by VladimirSirbu** [comment]
Great ! And what FPS it had when you run detection ?
For example, I want to detect real-time by camera, and I need good FPS even.

**Reply by stephanecharette** [reply]
Jetson Nano won't give you much.  It is more-or-less a toy.
You can see the results here:  https://www.ccoderun.ca/programming/2021-10-16_darknet_fps/

Read to the very bottom of that page to understand the numbers.  The realistic numbers for Jetson Nano are < 5 FPS when processing 1920x1080 video using a 416x416 yolov4-tiny network.

**Comment by VladimirSirbu** [comment]
So, on my computer I have just AMD Radeon Graphics (GPU). I think it isn't enough to run CUDA for example.

**Comment by ricchagas** [comment]
Isn't impossible train in CPU, but will take a lot of time . Other alternative is use Google colab with GPU,

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ricchagas** [comment, marked as answer]
In my use case I'm using a raspberry pi with a neural stick and I got about 7 fps . But depending of your input video resolution.

For Jetson there are a Nvidia sdk called deep stream , it's support yolo out of box ,

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
